.circleci/config.yml

```diff
@@ -199,7 +199,7 @@ jobs:
       - checkout
       - run:
           name: Install Python dependencies
-          command: python3.8 -m pip install -r requirements.txt
+          command: python3.8 -m pip install -r requirements/common.txt -r requirements/tests.txt
```

Comment:
you can have a `-r common.txt` line in `requirements/tests.txt` so that everything in common.txt is installed automatically when installing tests.txt

Reply:
also changed for docs and added minimum.txt